layouts/shortcodes/admin-scim.html

```diff
@@ -67,6 +70,26 @@
 3. In the SSO connections table, select the **Actions** icon.
 4. Select **Disable SCIM**.
 
-## Limitations
+## Set up role mapping
 
-Administrators can assign [roles](/docker-hub/roles-and-permissions/) to organization members. However, SCIM doesn't support role management.
+You can assign [roles](/docker-hub/roles-and-permissions/) to members in your organization in the IdP. To set up a role, you can use optional user-level attributes for the person you want to assign a role. In addition to roles, you can set an organization and team to override the default provisioning for the SSO connection.
+
+These mappings are supported for both SCIM and JIT provisioning.
+
+The optional user-level attributes are:
+
+- `dockerRole` - The available roles to assign are `member`, `editor`, or `owner`. For a list of permissions for each role, see [Roles and permissions](/docker-hub/roles-and-permissions/).
+- `dockerOrg` - Setting this attribute overrides the default organization configured for the SSO connection, and won't add them to the default team. If this attribute isn't set, the user is provisioned to the default organization and the default team.
+- `dockerTeam` - Setting this attribute provisions the user to the default org and to the specified team, instead of the SSO connection's default team. This will also create the team if it doesn't exist. You can still use group mapping to provision users to teams in multiple orgs. See {{ $group_link }}.
```

Comment:
If both `dockerOrg` and `dockerTeam` are set, it will provision the user to the team within that org

Reply:
I think I'll update this to be a table to make it easier to read the different scenarios. QQ what's the format for the value that can be used for `dockerOrg` and `dockerTeam`? Is it it `organization` and `organization:team`?

Reply:
Just `organization` and `team` (Colon version is only need in group mapping due to multi-org)